lib/scrabble.rb

```diff
@@ -1,7 +1,36 @@
 class Scrabble
 
+  def standardized_format(word)
+    word.upcase
+  end
+
   def score(word)
-    1
+    return 0 if word.nil?
+
+    values = point_values
+    word = standardized_format(word)
+
+    sum = 0
+    word.each_char do |char|
```

Comment:
We'd love to see you pushing to use more advanced enums, like reduce here. It's not expected quite yet, but the sooner you practice, the better you'll become.